### PR TITLE
Handling empty generations returned by Gemini 1.5 Pro API

### PR DESCRIPTION
**Pull request by Vaibhav2001**
# Pull Review Summary

## Description

Handling empty generations returned by Gemini 1.5 Pro API

## Type of change

- Bug fix (change which fixes an issue)

## Tests

### CODE REVIEW

File: src/autolabel/models/google.py [hunk 1]
```diff
@@ -83,6 +82,17 @@ async def _alabel(self, prompts: List[str]) -> RefuelLLMResult:
             result = await self.llm.agenerate(prompts)
             generations = result.generations
             end_time = time()
+            if generations == [[]]:
```
Comment (rajasbansal) [inline src/autolabel/models/google.py:85]: can we make this more generic

File: src/autolabel/models/google.py [hunk 2]
```diff
@@ -1,11 +1,10 @@
 import os
 import logging
-from functools import cached_property
 from time import time
 from typing import List, Optional
 
-from langchain.schema import Generation, HumanMessage, LLMResult
-from tenacity import before_sleep_log, retry, stop_after_attempt, wait_exponential
+from langchain.schema import Generation
+from tenacity import before_sleep_log, retry
```
Comment (rajasbansal) [inline src/autolabel/models/google.py:7]: is this used anywhere?

**Review by rajasbansal — APPROVED** [review]

File: src/autolabel/models/google.py (hunk 2 above)
Comment (Vaibhav2001) [reply src/autolabel/models/google.py:7]: yeah i do not think so.

File: src/autolabel/models/google.py (hunk 1 above)
Comment (Vaibhav2001) [reply src/autolabel/models/google.py]: done

**Review by rajasbansal — APPROVED** [review]

**Pull request merged**